syntax = "proto2";

import "google/protobuf/descriptor.proto";
package bgs.protocol;

option java_package = "bnet.protocol";
option java_outer_classname = "ServiceOptionsProto";
option optimize_for = CODE_SIZE;

message BGSServiceOptions {
  optional string descriptor_name = 1;
}

message SDKServiceOptions {
  optional bool inbound = 1;
  optional bool outbound = 2;
}

extend .google.protobuf.ServiceOptions {
  optional .bgs.protocol.BGSServiceOptions service_options = 90000;
  optional .bgs.protocol.SDKServiceOptions sdk_service_options = 90001;
}


